docs/standard/class-libraries.md

```diff
@@ -49,6 +49,7 @@ The .NET Standard does not expose the entire functionality of the .NET Framework
 The following platforms support .NET Standard libraries:
 
 *   .NET Core
+*   Universal Windows Platform (UWP), Fall Creators Update or newer
```

Comment:
should we simply list the version here instead?
Universal Windows Platform (UWP) 10.0 and later versions

@terrajobst?